.agents/skills/theme-creator/SKILL.md

```
@@ -0,0 +1,86 @@
+
+### 2. Injetar o CSS
+Adicione o novo bloco `:root[data-theme='novo-nome-tema']` no final do arquivo `src/styles/themes.css`.
+
+### 3. Registrar o Tema no TypeScript
+Modifique o arquivo `src/utils/theme.ts`.
+Encontre o array `SITE_THEMES`:
+`export const SITE_THEMES = ['zen-night', 'mediterranean-dusk'] as const;`
+Adicione o novo nome do tema ao final do array. Exemplo:
+`export const SITE_THEMES = ['zen-night', 'mediterranean-dusk', 'halloween-night'] as const;`
+
+### 4. Validação Final
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)